cmd/checkcommits/checkcommits.go

```diff
@@ -467,6 +467,13 @@ func detectCIEnvironment() (commit, dstBranch, srcBranch string) {
 			// contains that branch: master doesn't exist.
 			dstBranch = "origin"
 		}
+	} else if os.Getenv("CI") != "" {
```

Comment:
I'm not sure `CI` is the most accurate thing we can check for Jenkins. There are some more Jenkins specific env vars I think: https://wiki.jenkins.io/display/JENKINS/Building+a+software+project
Including:
- `WORKSPACE`
- `JENKINS_URL`
- or we could even check we have some/one/all of the `ghprb` vars set?